starknet_py/cairo/serialization/data_serializers/uint256_serializer.py

```diff
@@ -0,0 +1,45 @@
+from typing import Generator
+
+from starknet_py.cairo.felt import uint256_range_check
+
+from starknet_py.cairo.serialization._context import (
+    DeserializationContext,
+    SerializationContext,
+)
+from starknet_py.cairo.serialization.data_serializers.cairo_data_serializer import (
+    CairoDataSerializer,
+)
+
+U128_UPPER_BOUND = 2**128
+
+
+class Uint256Serializer(CairoDataSerializer[int, int]):
+    """
+    Serializer of Uint256. In Cairo it is represented by structure {low, high}.
+    Can serialize an int.
+    Deserializes data to an int.
+    """
+
+    def deserialize_with_context(self, context: DeserializationContext) -> int:
+        [low, high] = context.reader.read(2)
+
+        # Checking if resulting value is in [o, 2^256) range is not enough. Uint256 should be made of two uint128.
```

Comment:
```suggestion
        # Checking if resulting value is in [0, 2^256) range is not enough. Uint256 should be made of two uint128.
```
Any reason why it is an "o" letter?